gsfpy/bindings.py

```diff
@@ -25,7 +25,21 @@
 
 _gsf_lib_rel_path = "libgsf/libgsf03-08.so"
 _gsf_lib_abs_path = path.join(path.abspath(path.dirname(__file__)), _gsf_lib_rel_path)
```

Comment:
Obviously not necessary, but I think it's a good change to make since you're in the area

```suggestion
from pathlib import Path

_gsf_lib_abs_path = str(Path(__file__).parent / "libgsf" / "libgsf03-08.so")
```

Reply:
Done